pypesto/predict/task.py

```diff
@@ -28,7 +28,7 @@ class PredictorTask(Task):
 
     def __init__(
         self,
-        predictor: 'pypesto.predict.Predictor',  # noqa: F821
+        predictor,  #: 'pypesto.predict.Predictor',  # noqa: F821
```

Comment:
is this also commented due to circular imports?

Reply:
No, because it doesn't exist :expressionless: . This whole module is not used anywhere. I think it could/should be removed, but went for the more conservative approach.

Reply:
Ah, it should be `pypesto.predict.AmiciPredictor` I guess -- there is no parent `Predictor` class.

Reply:
Possible, but not sure what was intended there, as it's really not used anywhere.

Reply:
Was introduced in this monster, without usage https://github.com/ICB-DCM/pyPESTO/pull/490/